sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java

```diff
@@ -65,19 +65,24 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
```

Comment:
Is it possible to add a new unit test that would break for the previous implementation (due to pane numbers being incorrect) but would pass for the new implementation ?

Reply:
possibly - trying to think how to best make that trigger.